frontend/plan/actions/index.js

```diff
@@ -172,7 +172,7 @@ export const clearSchedule = () => ({
 });
 
 export const loadRequirements = () => (dispatch) =>
-    doAPIRequest("/requirements/").then(
```

Comment:
do we never use `doAPIRequest` anymore? it also sets some headers/cookies that we need, so I think the answer is to modify `doAPIRequest` to not default to `/api/plan` but simply `/api`, rather than circumvent it altogether.

Reply:
Yeah that makes sense